invenio_oauthclient/contrib/orcid.py

```diff
@@ -170,7 +170,11 @@ def account_info(remote, resp):
     return dict(
         external_id=orcid,
         external_method='orcid',
-        user=dict()
+        user={
+            'profile': {
+                'full_name': resp.get('name')
```

Comment:
Can you add commas at the end of line after which new item can be added?

(175, 176, 177)

--
Can we use `{}` instead of `dict()` also at 170? I see that it's used inconsistently in the file (not your fault, but we should improve it when touching the file). Thanks!